### DISCUSSION TITLE
wrong ServiceAccount properties to initializeApp for admin

### DISCUSSION THREAD

**Discussion by hendrik244** [gladly-team/next-firebase-auth#172] [Q&A]
I tried to try the example.
During the compilation i the following error:
```
errorInfo: {
    code: 'app/invalid-credential',
    message: 'Service account object must contain a string "private_key" property.'
  },
```

My .env.loval file looks like this:

FIREBASE_PRIVATE_KEY=-----BEGIN PRIVATE KEY-----XXXXXXXXXXX-----END PRIVATE KEY-----

I think it's because the interface is containing the wrong json properties(?):

`firebaseAdminInitConfig?: {
    credential: {
      projectId: string; --> should be "project_id"
      clientEmail: string; --> should be "client_email"
      privateKey: string; --> should be "private_key"
    };
    databaseURL: string;
  };`

**Comment by shreyas-jadhav** [comment]
Have you tried wrapping the FIREBASE_PRIVATE_KEY with single quotes and then with double quotes, for example

`FIREBASE_PRIVATE_KEY="'----BEGIN.... ---'"`

**Reply by kmjennison** [reply]
With this approach, note that you'll have to use `JSON.parse`. See docs:
https://github.com/gladly-team/next-firebase-auth#firebaseadmininitconfig
